src/mono/wasm/debugger/BrowserDebugProxy/SignatureTypeProvider.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.WebAssembly.Diagnostics;
+
+namespace Microsoft.WebAssembly.Diagnostics
```

Comment:
File-scoped namespace:
```suggestion
namespace Microsoft.WebAssembly.Diagnostics;
```